### PR TITLE
bm ifnextchar update

### PR DESCRIPTION
**Pull request by davidcarlisle**
## Checklist of required changes before merge will be approved
- [X] Test file(s) added
- [X] Version and date string updated in changed source files
- [X] Relevant `\changes` entries in source included
- [X] Relevant `changes.txt` updated
- [X] `ltnewsX.tex` updated

### CODE REVIEW

File: base/doc/ltnews33.tex [hunk 1]
```diff
@@ -696,6 +696,14 @@ \subsection{\pkg{longtable}: General bug fix update}
 
 
 
+\subsection{\pkg{bm}: Better support for commands with optional arguments}
+Some uses of optional arguments that were supported by \cs{bm} stopped
+being supported (in 2004) when \cs{kernel@ifnextchar} was used
+internally by the format instead of \cs{@ifnextchar} This update
```
Comment (FrankMittelbach) [inline base/doc/ltnews33.tex:702]: period missing

**Review by FrankMittelbach — APPROVED** [review]
looks fine to me with minor comment.

Are you also going for #71 while you have that file being changed?

File: base/doc/ltnews33.tex (hunk 1 above)
Comment (davidcarlisle) [reply base/doc/ltnews33.tex]: @FrankMittelbach  I added the . I'll have to think about 71 hold off on the PR for a bit (I'll flip it to draft)

**Comment by davidcarlisle** [comment]
@FrankMittelbach  I added a section on the new package options, so I think this is good to merge now

**Comment by car222222** [comment]
So this one is done!  Good news and thanks.

**Pull request merged**